third_party/pcl.BUILD

```diff
@@ -476,7 +476,7 @@ pcl_library(
         ":common",
         ":kdtree",
         ":octree",
-        # TODO flann. Seems to work anyway?
+        "@flann//:flann",
```

Comment:
Similar here, you can just use `@flann` instead of `@flann//:flann` since you are using the default target.